atlasdb-backup/src/main/java/com/palantir/atlasdb/backup/AtlasBackupService.java

```diff
@@ -81,6 +104,7 @@ private void storeBackupToken(InProgressBackupToken backupToken) {
         CompleteBackupResponse response = atlasBackupClientBlocking.completeBackup(authHeader, request);
 
         return response.getSuccessfulBackups().stream()
+                .filter(coordinationServiceRecorder::verifyFastForwardState)
```

Comment:
Don't forget we also have to actually persist the coordination service state at fast forward timestamp along with the other backup state